### PR TITLE
modifications for standalone - hit selection windows

### PR DESCRIPTION
**Pull request by leonardogiannini**
some corrections to the standalone code used for deriving the hit selection windows

### CODE REVIEW

File: RecoTracker/MkFitCore/src/MkFinder.cc [hunk 1]
```diff
@@ -611,19 +614,21 @@ namespace mkfit {
...
                       "h_ex/F:h_ey/F:h_er/F:h_ephi/F:h_ez/F:"
                       "ht_dxy/F:ht_dz/F:ht_dphi/F:"
-                      "h_chi2/F"
+                      "h_chi2/F:bkwd/I"
                       "\n");
                   first = false;
                 }
 
-                if (!(std::isnan(phi)) && !(std::isnan(getEta(m_Par[iI].At(itrack, 5, 0))))) {
+		if (1 && !(std::isnan(phi)) && !(std::isnan(getEta(m_Par[iI].At(itrack, 5, 0)))) && m_Label(itrack, 0, 0)>=0 && m_Label(itrack, 0, 0)==mchid && m_Label(itrack, 0, 0)==thisseedmcid )  {
```
Comment (slava77) [inline RecoTracker/MkFitCore/src/MkFinder.cc:622]: is `1 &&` needed?

Comment (leonardogiannini) [reply RecoTracker/MkFitCore/src/MkFinder.cc:622]: of course not

I need to put some other nan protections and the hit-track selection under ifdef

**Comment by slava77** [comment]
@leonardogiannini 
are you still working on this or is it worth to integrate to master/cmssw?

**Comment by leonardogiannini** [comment]
Maybe forcing the push to the old branch was not a good idea!
Anyway, these are the changes that we need https://github.com/trackreco/cmssw/pull/128/commits/e10bc031574cf7e36a5e6007fe6df98ba85816cf

**Comment by slava77** [comment]
> Maybe forcing the push to the old branch was not a good idea! Anyway, these are all the changes that we need [e10bc03](https://github.com/trackreco/cmssw/commit/e10bc031574cf7e36a5e6007fe6df98ba85816cf)

it looks like you merged master or smth instead of rebasing the existing branch
check http://cms-sw.github.io/tutorial-resolve-conflicts.html

**Comment by slava77** [comment]
> I had to move to a more recent release. trackreco/master is also behind master and I can't sync it

I've updated trackreco/master now, but the extra commits are still here; perhaps try to force-push once more?

**Comment by slava77** [comment]
a simple comparison https://github.com/trackreco/cmssw/compare/master...leonardogiannini:cmssw:windows-fix?expand=1 doesn't show any extra commits.
I'm not sure if doing a resync after the update triggered the current state here.

It may be easier to open a new PR.

**Pull request closed without merging**